i18n/management/utils.py

```diff
@@ -8,19 +8,17 @@
 
 from django_slack import slack_message
 
-from i18n.models import Internationalizable
+from i18n.models import Internationalizable, InternationalizablePage
 
 
 def should_sync_model(model):
     """
     whether or not the given Django model is something the i18n sync should
     process. We care about models that:
-        - extend the Internationalizable model defined by this module
-        - are not proxy models (used by Django Admin for editing)
+        - directly extend the Internationalizable or InternationalizablePage
+          model defined by this module
     """
-    is_internationalizable = issubclass(model, Internationalizable)
-    is_not_proxy = not model._meta.proxy # pylint: disable=protected-access
-    return is_internationalizable and is_not_proxy
+    return Internationalizable in model.__bases__ or InternationalizablePage in model.__bases__
```

Comment:
I don't love this approach; what if we instead change the "is not proxy" rule to just be "does not extend the Django Admin model"

Reply:
What's the proper way to detect this? Something like `UnitStandard` is an admin proxy model, but only inherits from the `Unit` model:

https://github.com/mrjoshida/curriculumbuilder/blob/57cad8f62e50b03e4f16bf77cd9e2e1da5c3e44e/curricula/admin.py#L86-L90

Reply:
Oh, good point.

Hmm, I'm not sure what's the best way to do this, then. Would something like a `translate_proxy` flag be too gross? Something like

```python
def should_translate(model):
  if not issubclass(model, Internationalizable):
    return False

  if model._meta.proxy:
    return model._meta.translate_proxy or False

  return True
```

Reply:
We could, but I'm not sure I like that better than checking if it it's a direct subclass of one of the I18n models?

Reply:
I really don't like the idea of making _direct_ subclass into a thing that matters; seems then like we want mixins or some implementation that works outside of the class inheritance hierarchy.

To me, it seems like this logic is consistent with Django's underlying philosophy. Proxy models are virtual models that shouldn't matter, so we internationalize everything except proxy models, except we also allow individual proxy models to override that default behavior if there's a special reason.

Reply:
Updated in commit 242912797d7a0377f65e666af4008396e921c9d7! I couldn't add it to the class's Meta because I got:

```
TypeError: 'class Meta' got invalid attribute(s): translate_proxy
```